@Types/ITechRecord.d.ts

```diff
@@ -82,54 +85,60 @@ interface Axle {
     };
 }
 
-export interface AdrDetails {
```

Comment:
why did you remove all the interfaces on adrDetails?

Reply:
there is an interface for AdrDetails following the latest api specs, look below you'll find it